testsuite/MDAnalysisTests/coordinates/test_memory.py

```diff
@@ -75,6 +75,8 @@ def reader(self, trajectory):
 
     def iter_ts(self, i):
         ts = self.universe.trajectory[i]
+        # correct time because memory reader doesn't read the correct time
+        ts.time = ts.frame * self.dt
```

Comment:
Can you briefly clarify why DCD change is impacting memory reader here?

Reply:
the memory reader tests are based on the DCD file and assumed it started from 0. This code restores that assumption. The memory reader can't actually deal with a trajectory that doesn't start at 0 at the moment. See the new issue #1769